src/Command/UpdateAppPropertyCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DeveloperPortal\Cli\Command;
+
+use Keboola\DeveloperPortal\Cli\Command;
+use Keboola\DeveloperPortal\Exception;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class UpdateAppPropertyCommand extends Command
+{
+    protected function configure(): void
+    {
+        $this
+            ->setName('update-app-property')
+            ->addArgument('vendor', InputArgument::REQUIRED, 'Vendor ID')
+            ->addArgument('app', InputArgument::REQUIRED, 'App ID')
+            ->addArgument('property', InputArgument::REQUIRED, 'Name of the property to update')
+            ->addArgument('value', InputArgument::REQUIRED, 'Value of the property to update')
+            ->addOption('is-file', null, InputOption::VALUE_REQUIRED, 'Read value from file')
```

Comment:
Tohle se mi moc nezdá, udělal bych to jako to má třeba `docker run`, `--value` a `--value-file`?

![image](https://user-images.githubusercontent.com/497675/49698741-1d652900-fbc8-11e8-806e-6687164ca7cf.png)

Reply:
no jo, ale pak budu muset resit, co se stane kdyz napises:

`update-app-property shortDescription --value=test --value-file=readme.md` :)

Reply:
no tak `update-app-property shortDescription --value=shortDesc.md` nevyresim, to je jedno jestli napises `update-app-property shortDescription shortDesc.md` misto `update-app-property shortDescription --value=shortDesc.md --is-file` nebo `update-app-property shortDescription --value=shortDesc.md` misto `update-app-property shortDescription --value-file=shortDesc.md` a na `update-app-property shortDescription --value="my description" --is-file` to hodi ze `file "my description" does not exist`. 

no jeste o tom budu chvilku premejslet anebo se treba najde nekdo s nejakym dalsim navrhem :)

Reply:
a co kdyby se to samo podívalo, jestli to není soubor?

Reply:
Mě přijde OK, že to jsou argumenty prostě to musíš zavolat `update-app-property vendor app property value`. Tím, že argument "value" je (logicky) required, tak nemůže být option. A tím pádem nejde IMHO udělat `--value` a `--value-file`. Tím pádem je možné buď switch (tj `--value-is-file`) nebo optional override (`--value-from-file=filename`), které by přepsalo hodnotu nehledě na to co je ve value. Ale ten switch mi přijde jako lepší řešení.

Reply:
>  že argument "value" je (logicky) required, tak nemůže být option

no nejak mi to tak prislo prirozene, ale neumim to tak hezky podat :)

@ondrejhlavacek 
> a co kdyby se to samo podívalo, jestli to není soubor?

jakoze `update-app-property shortDescription license` vlozi do short description license file? nevim no, zda se mi to dost magic

Reply:
> > a co kdyby se to samo podívalo, jestli to není soubor?
>
> jakoze update-app-property shortDescription license vlozi do short description license file? nevim no, zda se mi to dost magic

Souhlas. Jedině explicitně. Celé to načtení ze souboru je jen nice-to-have. Cokoli se dá s dostatkem snahy napsat do cli... 

Přemýšlel jsem, jestli neexistuje "in the wild" nějaká implementace něčeho podobného. Ale napadl mě jen `curl -d @filename.json`.  A to je zrovna debilní příklad. Ale přemýšlel jsem jestli by se to nemohlo číst ze stdin... a mohlo by to být "pokud chybí value, čti ji z STDIN"...

Reply:
no tim curlem jsem zacal a pak jsem to smazal, protoze se me to zdalo debilni ;)

stdin je travis unfriendly, kdyz tam bude chybet value (treba z envu), tak se 'sekne' build, protoze to bude cekat na stdin

Reply:
No, tak já myslel spíš `cat some.json | ... `

Reply:
hmm, nevim, mozna jo no (jestli to jde udelat)
co kdybysme zkusili treba jeste @yustme jestli nema nejakej napad :)

Reply:
@yustme jinymy slovy navrhujes to samy co najlos a tohle:

> Tím, že argument "value" je (logicky) required, tak nemůže být option. 

ti taky nevadi :)

ok, udelam to teda asi pres `--value` a `--value-from-file`

Reply:
@odinuv no value|value-from-file je required. Na to se musí udělat solo logika. To asi definice nepobere.